client/signup/steps/woocommerce-install/step-store-address/index.tsx

```diff
@@ -83,49 +97,85 @@ export default function StepStoreAddress( props: WooCommerceInstallProps ): Reac
 					<TextControl
 						label={ __( 'Address line 1' ) }
 						value={ get( WOOCOMMERCE_STORE_ADDRESS_1 ) }
-						onChange={ ( value ) => update( WOOCOMMERCE_STORE_ADDRESS_1, value ) }
+						onChange={ ( value ) => {
+							update( WOOCOMMERCE_STORE_ADDRESS_1, value );
+							clearError( WOOCOMMERCE_STORE_ADDRESS_1 );
+						} }
+						className={ address1Error ? 'is-error' : '' }
 					/>
+					<ControlError error={ address1Error } />
```

Comment:
For anyone that comes along, this is not how we should be doing error handling / validation. It sort of works but we really need a component that wraps the error and the text control together. What we might want to look at is updating the `form-input` components we already use in calypso to make use of the core component `TextControl` but that might not be doable. cc @SaxonF re library thinkings